catalog/imaging/imaging-transformer-nitf/src/main/java/org/codice/alliance/transformer/nitf/image/NitfImageTransformer.java

```diff
@@ -157,6 +174,11 @@ protected void handleComments(Metacard metacard, List<String> comments) {
         }
     }
 
+    private static void handleImageDateAndTime(ImageSegment imagesegmentHeader,
+            List<Date> imageDateAndTimeList) {
+        imageDateAndTimeList.add(convertNitfDate(imagesegmentHeader.getImageDateTime()));
```

Comment:
Can we just move this line up into handleImageSegmentHeader and remove this method?